docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+	{
+		"name": "EventStructWrongData",
+		"summary": "Using structs in events logged wrong data.",
+		"description": "If a struct is used in an event, the address of the struct is logged instead of the actual data.",
```

Comment:
It seems like I forgot "indexed" here in the original commit..

Reply:
But wasn't that irrelevant? (see #4921)